discovery-infra/test_infra/assisted_service_api.py

```diff
@@ -419,11 +409,8 @@ def create_client(
         url,
         offline_token=utils.get_env('OFFLINE_TOKEN'),
         pull_secret="",
-        wait_for_api=True,
         timeout=consts.WAIT_FOR_BM_API
 ):
     log.info('Creating assisted-service client for url: %s', url)
     c = InventoryClient(url, offline_token, pull_secret)
```

Comment:
And on kubeapi flow, why getting into client creation in the first place?

Reply:
it won't, that is something Igal changed recently, seems like it was merged today